bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/GuideGroup.scala

```diff
@@ -123,7 +123,7 @@ case class GuideGroup(grp: GuideGrp) extends java.lang.Iterable[GuideProbeTarget
           val opts = primary.fold(noPrimary) { t =>
             val (lefts, focusRight) = nel.toList.span(_ != t)
             focusRight.headOption.fold(noPrimary) { _ =>
-              OptsList(Zipper(lefts.toStream, t, focusRight.drop(1).toStream).right)
+              OptsList.focused(lefts, t, focusRight.drop(1))
```

Comment:
This is good.  Later, let's also switch to `OptsList.unfocused` for constructing the `noPrimary` case.